src/openms/source/ANALYSIS/SVM/SimpleSVM.cpp

```diff
@@ -202,6 +231,7 @@ void SimpleSVM::predict(vector<Prediction>& predictions, vector<Size> indexes) c
   }
 }
 
+// only works in classification mode
```

Comment:
In principle this should work for SVR too, as long as a linear kernel is used (otherwise the weights are meaningless). Is the implementation specific to classification?